frontend/pages/campaigns/index.tsx

```diff
@@ -51,9 +38,24 @@ const Campaigns = () => {
       </Head>
 
       <Container sx={containerSx} component="main">
-        <CampaignSearch searchDrawerIsOpen={searchDrawerIsOpen} setSearchDrawerIsOpen={setSearchDrawerIsOpen} />
+        <CampaignSearch
```

Comment:
![image](https://user-images.githubusercontent.com/53614454/196027769-228d408b-d410-48de-8ecf-6d10d7260492.png)

![image](https://user-images.githubusercontent.com/53614454/196027815-d1405652-fe6d-4b0e-b965-95b651d1c336.png)

Can we move the buttons down? Seems like is the convention for this type of forms.